chalicelib_smaht/checks/wfr_checks.py

```diff
@@ -193,10 +193,10 @@ def md5runCGAP_status(connection, file_type="", start_date=None, **kwargs):
 
 
 @action_function(start_missing=True, start_not_switched=True)
-def md5runCGAP_start(connection, start_missing=True, start_not_switched=True, **kwargs):
+def md5runsmaht_start(connection, start_missing=True, start_not_switched=True, **kwargs):
```

Comment:
If `CGAP` is upcased, should `SMaHT` be? Maybe it doesn't matter. But consistency on things like that can make the case easier to remember.